_posts/2026-04-02_top_5_deudores.md

```
@@ -0,0 +1,23 @@
+tags: [Admin, Super Admin, Riesgo]
+comments: false
+author: Catalina Pinilla.
+---
+
+## Matriz de riesgo - Top 5 deudores.
+
+Verás un recuadro con los 5 deudores principales del cliente (mostrando exclusivamente nombre y RUT) al ingresar a la Matriz de Riesgo Cliente.
+
+Esto te permitirá obtener una idea rápida de su concentración y principales pagadores, asegurando que la información provenga solo de facturas de los últimos 12 meses.
+
+#### Aquí te mostramos cómo se ve:
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)